public-ui/src/app/views/quick-search/quick-search.component.html

```diff
@@ -33,11 +33,11 @@ <h1 id='db-title' class="primary-display ">Data Browser</h1>
           </span>
     </div><!-- .search -->
     <div class="iconlinks">
-      <div class="icons"><a href="{{allOfUsUrl}}/frequently-asked-questions/" alt="FAQs"><img src="/assets/icons/icons_faqs.png" /><span>FAQs</span></a>
+      <div class="icons" (click)="iconClickEvent('faq')"><img src="/assets/icons/icons_faqs.png"/><span>FAQs</span>
```

Comment:
Leaving the anchor and adding a click handler to act in addition to the link behavior would be ideal; this gives better built-in browser behavior e.g. ctrl+click

Reply:
Done. Thanks.